### ISSUE TITLE
newlines should always render as a space

### ISSUE THREAD

**Issue by kentcdodds** [mdx-js/mdx#412]
### Make it possible to render a space for newlines between markdown

Example input:

```markdown
[this is a link](https://google.com/a-very-long-link-that-takes-an-entire-line-by-itself)
[this is another link](https://google.com/which-is-also-a-very-long-link-that-takes-an-entire-line-by-itself)
```

This will be rendered as:

```html
<a href="https://google.com/a-very-long-link-that-takes-an-entire-line-by-itself">this is a link</a><a href="https://google.com/which-is-also-a-very-long-link-that-takes-an-entire-line-by-itself">this is another link</a>
```

The solution is to add a space to the end of the first line or place them on the same line with a space between them, but prettier formats it the way it is above if you have "proseWrap" set to "always" which I do so I can reasonably edit the markdown.

So I _think_ this is an issue with how mdx converts those. I'm not sure how to solve this problem... I would like to make it possible to configure MDX to add a newline (or maybe this is a bug that can be fixed?)

### Your environment

N/A

### Steps to reproduce

See above. Copy/paste that into [the demo](https://mdxjs.com/)

### Expected behaviour

I would like the newline to be interpreted as a space when two links are side-by-side (like it is for other things).

### Actual behaviour

No space is inserted.

### More examples:

```markdown
_italics_
**bold**
```

That gets converted to:

```html
<em>italics</em><strong>bold</strong>
```

But this:

```markdown
italics
bold
```

Gets converted to:

```html
italics bold
```

Which I think is correct. I'd basically just like that space to be applied in all cases.

**Comment by timneutkens** [comment]
Sounds like a bug to me, cc @johno what do you think.

**Comment by johno** [comment]
Yeah this looks like an MDX bug. I’ll add a failing test today and try and get a fix in.

**Comment by johno** [comment]
This is now published as `0.18.1`. Thanks for reporting and let us know if you encounter any other issues!

**Comment by kentcdodds** [comment]
Sweet! Thanks! 💯
